submissions/AnnaStoyano/popup/style.css

```diff
@@ -0,0 +1,199 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    font-family: 'Segoe UI', Tahoma, Verdana, sans-serif;
+}
+
+.icon{ 
+    width:55px;
+    height: 55px;
+    background: url(./img/apps.png) no-repeat;
+}
+
+header{

```

Comment:
Avoid styling by HTML tag.
Separate responsibilities: HTML tags for markup, CSS for styling and visual representing and ID for javascript.